src/main/java/uk/gov/hmcts/probate/services/submit/controllers/v2/CasesController.java

```diff
@@ -42,4 +48,15 @@
         return ResponseEntity.ok(casesService.getCase(applicationId.toLowerCase(), caseType));
     }
 
+    @ApiOperation(value = "Get case to CCD using case Id", notes = "Get case to CCD")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Case retrieval from CCD successful"),
+            @ApiResponse(code = 400, message = "Case retrieval from CCD successful")
+    })
+    @GetMapping(path = "/cases/ccd/{caseId}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Is it possible to calls this something like "/case?caseId=12345", using a query param rather than path variable - will this work?